firmware/firmware.md

```diff
@@ -9802,7 +9802,8 @@ void loop() {
 
 The library provides the following log handlers:
 
-`SerialLogHandler`
+- `SerialLogHandler`
+- Additional community-supported log handlers can be found further below.
```

Comment:
I'm not sure we want to be linking from official docs to community creations either, but a place for community libraries/projects/example applications would be a nice addition to the Docs.  I know we've talked about adding this soon, so maybe we should hold off on adding this for now to see what the format of this new section will look like.  I'll tag this with a new milestone of "community" so we can keep track of the PRs/issues that represent community additions.